translations/de/info/task_description.html

```diff
@@ -59,69 +57,69 @@
+        dann ist die Frage: wer in aller Welt bin ich? Ja, das ist das Räthsel!« So ging sie
+        in Gedanken alle Kinder ihres Alters durch, die sie kannte, um zu sehen, ob sie in
+        eins davon verwandelt wäre.
     </p>
 
     <p>
-        "I'm sure I'm not Ada," she said, "for her hair goes in such
-        long ringlets, and mine doesn't go in ringlets at all; and I'm
-        sure I can't be Mabel, for I know all sorts of things, and she,
-        oh! she knows such a very little! Besides, SHE'S she, and I'm I,
-        and--oh dear, how puzzling it all is! I'll try if I know all the
-        things I used to know. Let me see: four times five is twelve,
-        and four times six is thirteen, and four times seven is--oh dear!
-        I shall never get to twenty at that rate! However, the
-        Multiplication Table doesn't signify:...
+        »Ich bin sicherlich nicht Ida,« sagte sie, »denn die trägt lange Locken, und mein
+        Haar ist gar nicht lockig; und bestimmt kann ich nicht Clara sein, denn ich weiß
+        eine ganze Menge, und sie, oh! sie weiß so sehr wenig! Außerdem, sie ist sie selbst,
+        und ich bin ich, und, o wie confus es Alles ist! Ich will versuchen, ob ich noch Alles
+        weiß, was ich sonst wußte. Laß sehen: vier mal fünf ist zwölf, und vier mal sechs
+        ist dreizehn, und vier mal sieben ist – o weh! auf die Art komme ich nie bis zwanzig!
+        Aber, das Einmaleins hat nicht so viel zu sagen; ...
     </p>
 
     <p style="float: right">
-        "Alice's Adventures in Wonderland." Lewis Carroll
+        "Alice's Abenteuer im Wunderland" von Lewis Carroll<br>
+        Aus dem Englischen von Antonie Zimmermann, 1869
     </p>
 
     <p style="clear: both;"></p>
 
 </div>
 
 <p>
-    After reading "Alice's Adventures in Wonderland,"
-    our robots decided to create their own "Multiplication table."
-    Stephan would lead this mission (yeah, that probably was a bad idea).
-    He forgot how to do multiplication and tried to invent a new method.
-    It’s a rather strange method if we may be so blunt.
+    Nach der Lektüre von "Alice im Wunderland ", haben sich unsere Roboter entschieden,
+    ihr eigenes Einmaleins zu erstellen. Stephan wollte diese Mission leiten
+    (ja, das war wahrscheinlich eine schlechte Idee). Er hat allerdings vergessen wie man
+    multipliziert und dann versucht eine neue Methode zu erfinden.<br>
+    Es ist eine ziemlich seltsame Methode, wenn wir so direkt sein wollen.
+</p>
+
+<p>
+    In Stephans Version der Multiplikation werden Zahlen zunächst in ihre binäre Darstellung
+    ohne führende Nullen konvertiert. Dann wird die erste Zahl vertikal geschrieben
+    (von oben nach unten) und die Zweite horizontal (von links nach rechts).
+    Damit füllen wir je eine Wertetabelle für die binären Operationen AND, OR und XOR,
+    sodass wir am Ende drei Tabellen haben.
```

Comment:
Your describtion of the notation is unclear and actually wrong.

> Damit füllen wir je....

We don't fill the table with the previous mentioned vertic- and horizontally written numbers. The results of the binary operations between the column values (horizontally number) and the row values fill these tables. Like the original states, where they cross each other tablewise.